.travis.yml

```diff
@@ -33,7 +33,7 @@ before_script:
 
 # Install composer dependencies
   - composer validate
-  - composer require --no-update silverstripe/recipe-cms:1.0.x-dev
+  - composer require --no-update silverstripe/recipe-cms:^4.0
```

Comment:
Stable will exclude test files so we'll need to keep using a dev constraint or switch to `--prefer-source`